README.md

```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
 
 ## Setup
 
+### Installing Python Using Anaconda
+1. Open https://www.anaconda.com/products/distribution with your web browser.
+2. Click on **Download** to download the Anaconda Installer. If you do not have enough space left on your computer, please refer to the section below: [Installing Python Using Miniconda](#installing-python-using-miniconda).
+3. Install Python 3 by running the Anaconda Installer using all of the defaults for installation. Make sure to check **Add Anaconda to my PATH environment variable**.
+4. You can check that Python is installed by opening a terminal and typing `python --version`. The response should be `Python 3.10.x`.
+5. Install the required packages by running the following lines in your terminal:
+```
+conda config --add channels conda-forge
+conda install voila=0.3.6
+```
+
+### Installing Python Using Miniconda
+1. Open https://docs.conda.io/en/latest/miniconda.html with your web browser.
+2. Click on the latest Installer link depending on your OS: **Miniconda3 Windows 64-bit** for Windows, **Miniconda3 macOS Intel x86 64-bit pkg** for MacOS, **Miniconda3 Linux 64-bit** for Linux.
+3. Install Python 3 by running the Anaconda Installer (double click on the downloaded file) using all of the defaults for installation. Make sure to check **Add Miniconda to my PATH environment variable**.
+4. You can check that Python is installed by opening a terminal and typing `python --version`. The response should be `Python 3.10.x`.
+5. Install the required packages by running the following lines in your terminal:
+```
+conda config --add channels conda-forge
+conda install jupyterlab=3.5.3 matplotlib=3.7.0 ipywidgets=7.6.5 voila=0.3.6 pandas=1.5.3
+```
+
```

Comment:
This is great. I would still suggest to add some instructions to check if everything is installed correctly (ie, just like you do with Python but for Jupyter and for importing the libraries within Jupyter).

Reply:
Maybe also would be worth adding a link to Software Carpentry installation instructions for troubleshooting?

Reply:
@maikia hmm, the Software Carpentry setup instructions I know of are for [Installing Python Using Anaconda](http://swcarpentry.github.io/python-novice-gapminder/setup.html#installing-python-using-anaconda)... So, pointing to those would contradict our previous review [comment](https://github.com/mkcor/python-prog/pull/7/files#r1156397840).

Instead, I was going to suggest adding a subsection for testing the setup (e.g., see last point of https://datacarpentry.org/image-processing/setup/). The test could consist in launching JupyterLab and running a code cell (I know that students will be learning how to do so in the first episode of this tutorial, but, well) such as:

```py
import pandas as pd                                                            
                                                                               
table = pd.DataFrame(
    {'Time': [0, 1, 2, 3],
     'Emma': [0, 10, 20, 30]}
)  
table.plot();
```
They would copy-paste the above code snippet and check that a plot shows up right below the code cell.

@bmaz does that make sense?